### ISSUE TITLE
.DS_Store is imported by generated file

### ISSUE THREAD

**Issue by Ericnr** [mmkal/slonik-tools#220]
I'm using `@slonik/typegen@0.6.0` and this is happening
```ts
import {.DS_Store} from './.DS_Store'
import {CompleteTodo} from './CompleteTodo'
import {CreateTodo} from './CreateTodo'
import {InsertTodo} from './InsertTodo'
import {_pg_types} from './_pg_types'
import {createUser} from './createUser'

export {.DS_Store}
export {CompleteTodo}
export {CreateTodo}
export {InsertTodo}
export {_pg_types}
export {createUser}

export interface KnownTypes {
  .DS_Store: .DS_Store
  CompleteTodo: CompleteTodo
  CreateTodo: CreateTodo
  InsertTodo: InsertTodo
  _pg_types: _pg_types
  createUser: createUser
}

export const knownTypes: KnownTypes = {
  .DS_Store,
  CompleteTodo,
  CreateTodo,
  InsertTodo,
  _pg_types,
  createUser,
}
```

**Comment by Ericnr** [comment]
This does not always happen and I cannot reproduce it consistently

**Comment by mmkal** [comment]
@Ericnr the fix for this is in [v0.6.2](https://www.npmjs.com/package/@slonik/typegen/v/0.6.2) - let me know if you have any more problems.
